### ISSUE TITLE
No settings available

### ISSUE THREAD

**Issue by ischiros** [nextcloud/documentation#1895]
Hello,

I use Nextcloud 18.0.2 and have File access control installed (v1.8.1).

The documentation says "After installing the File Access Control app as described in Apps management navigate to the configuration and locate the File Access Control settings."
But I can't find this anywhere. I've browsed all the settings screens twice, and nothing looks like this image https://docs.nextcloud.com/server/17/admin_manual/_images/files_access_control_sample_rules.png found on https://docs.nextcloud.com/server/17/admin_manual/file_workflows/access_control.html

Is that a bug, or I'm not looking at the right place ?
Documentation could be more specific about the place were the settings are located.

**Comment by skjnldsv** [comment]
Hi, the app is named Workflow now.
You can find it at https://cloud.domain.com/settings/admin/workflow

**Comment by ischiros** [comment]
Thanks for the quick answer.

I don't find any "Worklow" app, nor "File worklows".
There is just a "Workflow external scripts" in the store. Is it this one ?
And I do have a "File access control" installed. Shall I uninstall it because Workflow replaces it ?

**Comment by juliushaertl** [comment]
> Hi, the app is named Workflow now.

The settings naming is "Flow".

**Comment by ischiros** [comment]
Ow, got it.
What you see when you get in the settings is much different fro the screenshot in the documentation : you only have a big red rectangular with "Block File access" in it. That's why I didn't find it.

**Comment by juliushaertl** [comment]
Yes, we should update the screenshots and documentation for that, although I probably won't have time for that soon.

**Comment by ischiros** [comment]
Sure.
Thanks for your work :)

While I'm here : the flow to block access is named "block access to a file" (at leat, it is named like that in French). But folders are concerned also (which is a good thing). It would be great to make it more clear if you can.

**Comment by juliushaertl** [comment]
Contributions are of course always very welcome :wink:

**Comment by zlmitche** [comment]
I'm unable to follow this section https://docs.nextcloud.com/server/stable/admin_manual/file_workflows/access_control.html#prevent-uploading-of-specific-files using NextCloud 18.0.1

How do i go about disabling uploads of certain file names/types.
